### PR TITLE
Include the logo on readthedocs

### PR DESCRIPTION
**Pull request by eric-wieser**
Since the blue shows up poorly against the background, this just uses a pure-white version of the logo.

### CODE REVIEW

**Comment by ktbarrett** [comment]
Closes #2846.

**Review by ktbarrett — APPROVED** [review]
Looks good. Thanks!

File: documentation/source/conf.py [hunk 1]
```diff
@@ -173,7 +173,7 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-# html_logo = None
+html_logo = '_static/cocotb-logo-white.svg'
```
Comment (ktbarrett) [inline documentation/source/conf.py:176]: This will make Black happy.

```suggestion
html_logo = "_static/cocotb-logo-white.svg"
```

**Review by cmarqu — APPROVED** [review]

**Review by imphil — APPROVED** [review]

**Comment by imphil** [comment]
Thanks @eric-wieser!

**Pull request merged**